Cargo.lock

```diff
@@ -4285,7 +4285,7 @@ checksum = "57c0d7b74b563b49d38dae00a0c37d4d6de9b432382b2892f0574ddcae73fd0a"
 [[package]]
 name = "payjoin"
 version = "0.24.0"
-source = "git+https://github.com/payjoin/rust-payjoin.git?branch=master#5ecfed745bd33f6d5706f497bdb7662a3a7c9177"
+source = "git+https://github.com/payjoin/rust-payjoin.git?branch=master#ce0c7716be1d5e24168960ee824dc1b69359f943"
```

Comment:
I'm assuming we need to point to https://github.com/payjoin/rust-payjoin.git?branch=master#f3ded25b1609442a9e60621aba681f223e0db89a for the outptuts followup?

Reply:
Yeah we can do that outside this PR. But IIRC we need to restore outputs for ledger signing to work